nexus/src/app/mod.rs

```diff
@@ -100,6 +101,8 @@ pub const MAX_MEMORY_BYTES_PER_INSTANCE: u64 = 256 * (1 << 30); // 256 GiB
 pub const MIN_DISK_SIZE_BYTES: u32 = 1 << 30; // 1 GiB
 pub const MAX_DISK_SIZE_BYTES: u64 = 1023 * (1 << 30); // 1023 GiB
 
+pub const MAX_SSH_KEYS_PER_INSTANCE: u32 = 10;
```

Comment:
Why 10? Perhaps it would be good to add a comment here with the reasoning behind this number.

Reply:
Thanks for pointing this out, I meant to update it. The limit is absolutely arbitrary which I should comment as such. It's 8 in main (and silently ignores any SSH key over 8 in your profile when you're transferring it over). I've had a few conversations about these limits. There are a few places they are relevant.

1. What's the maximum number of keys a user can create in their account?
2. What's the maximum number of keys we want to support transferring to the instance?

If 2 is less than 1 then there's an edge case for users who have large amounts of SSH keys that want to just send them all to the instance. In the near term it might actually be good just to unify these limits. Just limit it to say... 100 SSH keys per account/instance (which was actually suggested by @iliana [in chat](https://matrix.to/#/!oSMXTSrgambaNFEemN:oxide.computer/$LuhOLmLVhoQNI7unHloUI_C66po73zfGHQJUWwcQWnc?via=oxide.computer&via=unix.house&via=matrix.org))